cfg/LocalRef.h

```diff
@@ -0,0 +1,47 @@
+#ifndef SORBET_CFG_LOCALREF_H
+#define SORBET_CFG_LOCALREF_H
+
+#include "core/LocalVariable.h"
+
+namespace sorbet::cfg {
+class CFG;
+
+class LocalRef final {
+    int _id;
+
+public:
+    LocalRef() : _id(0){};
```

Comment:
Is there an invariant that the localVariables vector in the CFG always has `LocalVariable::noVariable` as its first element?

Reply:
Yup; see the CFG constructor.